pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerBase.java

```diff
@@ -466,7 +466,7 @@ public CompletableFuture<Void> reconsumeLaterAsync(Message<?> message, long dela
     public CompletableFuture<Void> reconsumeLaterAsync(
             Message<?> message, Map<String, String> customProperties, long delayTime, TimeUnit unit) {
         if (!conf.isRetryEnable()) {
-            return FutureUtil.failedFuture(new PulsarClientException("reconsumeLater method not support!"));
+            return FutureUtil.failedFuture(new PulsarClientException("reconsumeLater method not supported! Have you enabled retries in your consumer?"));
```

Comment:
It's better to change `supported` to `enabled`. We can provide a direct guide message about how to enable this, like "Please enable retry in ConsumerBuilder"

Reply:
I agree with the second part, but disagree with the first part: we are talking about the method, which is not supported (we can't enable `reconsumeLater`). How about this?

"reconsumeLater method not supported! Please set `enableRetry(true)` in your `ConsumerBuilder`"

Reply:
"Please enable retry in ConsumerBuilder to support `reconsumeLater`" ?

Reply:
@Technoboy- that doesn't sound good. One would say "enable retries" in English. Thus, I proposed "Please set `enableRetry(true)` in your `ConsumerBuilder`".

I think the Pulsar option should be named `enableRetries` instead, but that's a different story...

Reply:
@Jason918 @Technoboy- any thoughts?

Reply:
Instead of asking a question, I think it would be appropriate to return a statement with the name of the specific configuration value. It could be something like:

```suggestion
            return FutureUtil.failedFuture(new PulsarClientException("reconsumeLater method not supported becuase retryEnabled is false."));
```

Reply:
@michaeljmarshall that's a good idea (there's a typo in that commit suggestion, though 😄 ), but I would still add where retryEnabled is false (in the `ConsumerBuilder`), I think that'd make for a better error message.

Reply:
Sure, that makes sense, and good catch on my typo.